python_modules/libraries/dagster-dbt/dagster_dbt/cli/resources_v2.py

```diff
@@ -258,6 +274,49 @@ def build_asset_selection(
             exclude=dbt_exclude,
         )
 
+    def build_schedule(
+        self,
+        job_name: str,
+        cron_schedule: str,
+        dbt_select: str = "fqn:*",
+        dbt_exclude: Optional[str] = None,
+        tags: Optional[Mapping[str, str]] = None,
+        config: Optional["RunConfig"] = None,
+        execution_timezone: Optional[str] = None,
+    ) -> ScheduleDefinition:
+        """Build a schedule to materialize a specified set of dbt resources from a dbt selection string.
+
+        See https://docs.getdbt.com/reference/node-selection/syntax#how-does-selection-work.
+
+        Args:
+            job_name (str): The name of the job to materialize the dbt resources.
+            cron_schedule (str): The cron schedule to define the schedule.
+            dbt_select (str): A dbt selection string to specify a set of dbt resources.
+            dbt_exclude (Optional[str]): A dbt selection string to exclude a set of dbt resources.
+            tags (Optional[Mapping[str, str]]): A dictionary of tags (string key-value pairs) to attach
+                to the scheduled runs.
+            config (Optional[RunConfig]): The config that parameterizes the execution of this schedule.
```

Comment:
Not blocking, but it could be helpful to add some docstring here to help users understand what this "config" is and how it relates to the kind of dbt config they'd want to set.